src/query/service/Cargo.toml

```diff
@@ -103,7 +103,7 @@ metrics = "0.20.1"
 naive-cityhash = "0.2.0"
 once_cell = "1.15.0"
 opendal = { version = "0.21", features = ["layers-tracing", "layers-metrics", "compress"] }
-opensrv-mysql = "0.2.0"
+opensrv-mysql = { git = "https://github.com/datafuselabs/opensrv", rev = "5e37788" }
```

Comment:
How about bumping a new version for opensrv-mysql?

Reply:
Has discussed this with @PsiACE , and he recommends using rev first.